### ISSUE TITLE
error compiling hint-mixins.scss

### ISSUE THREAD

**Issue by sly7-7** [chinchang/hint.css#170]
In a ember app, using ember-cli-sass (with https://sass-lang.com/dart-sass as compiler), I have the following error during compilation:
```
Sass Syntax Error (SassCompiler) in /Users/sma01/work/git/gitlab/erdil/esatis/frontend/node_modules/html-hint/node_modules/hint.css/src/hint-mixins.scss:18:5

Error: Top-level selectors may not contain the parent selector "&".
                                &.#{$hintPrefix}#{$position}-#{$xDir}:before {
                                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  node_modules/html-hint/node_modules/hint.css/src/hint-mixins.scss 18:5                                                                                          arrow-border-color()
  node_modules/html-hint/node_modules/hint.css/src/hint-position.scss 81:1                                                                                        @import
  node_modules/html-hint/node_modules/hint.css/src/hint.scss 21:9                                                                                                 @import
  node_modules/html-hint/html-hint.sass 1:9
```

I just don't know what that mean.

**Comment by chinchang** [comment]
Seems like dart-sass has some implementation differences to node-sass because it's working fine with node-sass.
Have found the solution though -> https://github.com/sass/sass/issues/1873#issuecomment-152293725
Fixing this.

**Comment by chinchang** [comment]
fixed in 2.5.1
